doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -189,6 +219,30 @@ Deployment
     # Start a Superlink
     $ flower-superlink --insecure
 
+    # Flower v1.13
+    # In a new terminal window, start a long-running SuperNode
+    $ flower-supernode \
+         --insecure \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=0 num-partitions=2" \
+         --supernode-address 127.0.0.1:9094 \
+         --isolation subprocess
+
+    # In another terminal window, start another long-running SuperNode (at least 2 SuperNodes are required)
+    $ flower-supernode \
+         --insecure \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=1 num-partitions=2" \
+         --supernode-address 127.0.0.1:9095 \
+         --isolation subprocess
+
+    # In a new terminal, start the SuperExec process with the following command:
+    $ flower-superexec \
+        --insecure \
+        --executor flwr.superexec.deployment:executor \
+        --executor-config 'superlink="127.0.0.1:9091"'
```

Comment:
For 1.13, we have deprecated `flower-superexec`. The two commands users need to use are `flower-superlink` and `flower-supernode`.

Reply:
Thank you for pointing this out. Removed.